structuredefinitions/UKCore-DiagnosticReport-Lab.xml

```diff
@@ -45,7 +45,7 @@
         </discriminator>
         <rules value="open" />
       </slicing>
-      <min value="1" />
+      <max value="1" />
```

Comment:
This change has made Observation.category, go from 1..*, to 0..1
What you need to do is change this back, and apply a 1..1 cardinality on the sliceLaboratory